pyanaconda/modules/storage/devicetree/model.py

```diff
@@ -20,6 +20,7 @@
 import os
 
 from blivet.blivet import Blivet
+from blivet.blivet import flags as blivet_flags
```

Comment:
Just a nitpick: I think it would be "more correct" to have the import as `from blivet.flags import flags as...`. This way you are relying on `blivet/blivet.py` importing the flags which will probably never change, but it might, the correct path to the flags singleton should be `blivet.flags.flags`.

Reply:
Thanks, will change it.